packages/fxa-payments-server/src/store/actions/api.ts

```diff
@@ -51,10 +51,18 @@ export default {
         return { ...result, subscriptionId };
       },
     } as const),
-  reactivateSubscription: (subscriptionId: string) =>
+  reactivateSubscription: (subscriptionId: string, plan: Plan) =>
     ({
       type: 'reactivateSubscription',
-      payload: apiReactivateSubscription(subscriptionId),
+      meta: { plan },
+      payload: async () => {
+        // Ignore the API result, because it's just `{}` on success.
+        await apiReactivateSubscription(subscriptionId);
+        return {
```

Comment:
Oh yeah, and this is the return type that `ActionPayload` carries through into the Redux state.